kivy/metrics.py

```diff
@@ -158,8 +159,13 @@ def dpi(self):
             return float(custom_dpi)
 
         if platform == 'android':
-            import android
-            return android.get_dpi()
+            if USE_SDL2:
```

Comment:
I had to modify Hardware.java to make it work (maybe due to an android bug, http://stackoverflow.com/questions/27590213/android-displaymetrics-densitydpi-returns-zero-on-rooted-devices) and I didn't test that with the pygame bootstrap. I'll change it all to pyjnius if that works.